includes/wsuwp-shortcode-fundselector.js

```diff
@@ -7,31 +7,21 @@ jQuery(document).ready(function($) {
         {
 			source: function( request, response ) {
 				$("#fundSearch").addClass("loading");
-				$.getJSON( wpData.request_url_base + 'idonate_fund', 
```

Comment:
This would need to be modified slightly if the `filter[s]` parameter suggestion above is sufficient.